web/src/components/datasets/DatasetDetailPage.tsx

```diff
@@ -147,9 +170,31 @@ const DatasetDetailPage: FunctionComponent<IProps> = props => {
               />
             </Tabs>
           </Box>
-          <IconButton onClick={() => history.push('/datasets')}>
-            <CloseIcon />
-          </IconButton>
+          <Box display={'flex'} alignItems={'center'}>
+            <Box mr={1}>
+              <Button
+                color='primary'
+                className={classes.buttonDelete}
+                onClick={() => {
+                  props.dialogToggle('')
+                }}
+              >
+                {i18next.t('jobs.delete')}
+              </Button>
+              <Dialog
+                dialogIsOpen={display.dialogIsOpen}
+                dialogToggle={dialogToggle}
+                title={'Are you sure?'} // i18next.t('jobs.dialogTitleConfirm')
```

Comment:
Don't we wish have this in the internationalization configuration?

Reply:
Sure, forgot to add it in previous commit